subt/script/run_maria.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+(python3 -m subt.main run config/maria-ros.json --side left --speed 0.5 --walldist 0.7 --start-paused;python3 -m osgar.record config/test-lora.json) &
```

Comment:
I think that we used 0.6 at the end? (Beta Course) - but can be left to another PR

Reply:
Yes, you are right, we used 0.6.